R/model-tools.R

```diff
@@ -460,3 +464,178 @@ enw_stan_to_r <- function(
   }
   return(mod)
 }
+
+#' Set caching location for Stan models
+#'
+#' This function allows the user to set a cache location for Stan models
+#' rather than a temporary directory. This can reduce the need for model
+#' compilation on every new model run across sessions or within a session.
+#' For R version 4.0.0 and above, it's recommended to use the persistent cache
+#' as shown in the example.
+#'
+#' @param path A valid filepath representing the desired cache location.
+#'
+#' @param type A character string specifying the cache type. It can be one of
+#' "session", "persistent", or "all". Default is "session".
+#' "session" sets the cache for the current session, "persistent" writes the
+#' cache location to the user’s `.Renviron` file,  and "all" does both.
```

Comment:
I changed some of the wording around and made the recommendation be to use `c("session", "persistent")` rather than `"all"` as this felt more explicit.  As the user is going to copy and paste this felt okay.